modules/tinymce/changelog.txt

```diff
@@ -1,4 +1,5 @@
 Version 5.7.0 (TBD)
+    Fixed the order of CSS precedence of `content_style` and `content_css` in the `preview` and `template` plugins. `content_style` now has precedence #TINY-6529
```

Comment:
This isn't the correct placement. If you look at the Notion document I linked yesterday, fixed should be after changed.

- [ ] Fix placement of changelog

Reply:
oh, ok. It went by when I've fixed conflicts :-) fixing now, thanks @lnewson

Reply:
Done